packages/visx-text/src/Text.tsx

```diff
@@ -115,7 +115,7 @@ class Text extends React.Component<TextProps, TextState> {
     // Only perform calculations if using features that require them (multiline, scaleToFit)
     if (props.width || props.scaleToFit) {
       if (needCalculate) {
-        const words: string[] = props.children
+        const words: string[] = (props.children !== null && props.children !== undefined)
```

Comment:
this could be simplified to `props.children != null` which captures both conditions